test/models/user_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  
+  def setup
+    foo = ('a'..'z').to_a.shuffle.first(8).join # random 8char string for unique email
+    @user = User.new({:email => "#{foo}@gmail.com", :first_name => "someFirstName", 
+                      :last_name => "someLastName", :password => "111111", 
+                      :password_confirmation => "111111" }).save(:validate => false)
+    # save(:validate) is needed only when you use email validation, I think?
+      #no, actually you've got to have it. Gotta save, and gotta ( => false).
```

Comment:
Incorrect indentation detected (column 6 instead of 4).<br>Missing space after #.